### PR TITLE
enabled gzip compression

### PR DESCRIPTION
**Pull request by roman-vabishchevych**
## Related Issue
https://github.com/webiny/webiny-js/issues/626

### CODE REVIEW

File: packages/cli/create/template/apps/site/handler/handler.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const fs = require("fs");
 const path = require("path");
 const mime = require("mime-types");
 const isUtf8 = require("isutf8");
+const zlib = require('zlib');
```
Comment (Pavel910) [inline packages/cli/create/template/apps/site/handler/handler.js]: This looks like you've committed without running the commit hooks.  Please either format the file with prettier, or let the commit hooks do it for you. But don't use the `--no-verify` ;)

Comment (roman-vabishchevych) [reply packages/cli/create/template/apps/site/handler/handler.js]: done

**Comment by roman-vabishchevych** [comment]
@Pavel910 
prettified :)

**Comment by Pavel910** [comment]
I'll merge this a little bit later, busy with something. Thanks for the PR, looks good :) 🚀

**Pull request merged**